patientsearch/config.py

```diff
@@ -26,6 +26,30 @@ def load_json_config(potential_json_string):
     "Your account is not authorized for access, please contact an administrator",
 )
 
+DASHBOARD_COLUMNS = os.getenv(
+    "DASHBOARD_COLUMNS",
+    [
+        {
+            "label": "First Name",
+            "expr": "$.name[0].given[0]",
+        },
+        {
+            "label": "Last Name",
+            "expr": "$.name[0].family",
+        },
+        {
+            "label": "Birth Date",
+            "expr": "$.birthdate",
```

Comment:
@pbugni I think this should be `$.birthDate` ([FHIR patient reference](https://build.fhir.org/patient.html))

Reply:
i've found HAPI is inconsistent with the case of `birthDate`, anyhow, thanks for the request, I'm guessing it didn't work as was...